index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey-From</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="title">freeCodeCamp Survey Form</h1>
+    <p class="description"><i>Thank you for taking the time to help us improve the platform</i></p>
```

Comment:
Please create a header tag and put it inside this h1 and p tag, so look like professional and clean code 👍